client/src/templates/Introduction/components/block.tsx

```diff
@@ -78,9 +78,9 @@ export class Block extends Component<BlockProps> {
 
   renderCheckMark(isCompleted: boolean): JSX.Element {
     return isCompleted ? (
-      <GreenPass style={mapIconStyle} />
+      <GreenPass hushScreenReaderText style={mapIconStyle} />
```

Comment:
I am struggling to understand/find where this property comes from? Who sets it?

Reply:
It's just a boolean flag we created to inhibit the default behavior of the GreenPass icon (which is to add the screen reader only text for the icon). It it only used for the beta rwd right now, though I suppose it could be used for other new courses down the road. My thinking is that at some point we won't want the GreenPass icon to ever include the screen reader only text and then we can get rid of this flag.

Reply:
@ShaunSHamilton if I understand your question correctly, it's being set here because you don't need to bother specifying `hushScreenReaderText = { true }` just adding the prop is enough.